### PR TITLE
Update ticketing.md

### PR DESCRIPTION
**Pull request by Julien-Rozet**
## Description

Translations of menus, french words and typo.

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x
- [ ] 22.04.x (master)

### CODE REVIEW

File: fr/alerts-notifications/ticketing.md [hunk 1]
```diff
@@ -35,63 +34,64 @@ configuration.
...
 
 Pour utiliser le widget, vous devez l'ajouter dans une vue
-personnalisée. Allez dans le menu `Home > Custom Views`, sélectionnez
-votre vue et cliquez sur le bouton **Add widget**.
+personnalisée. Allez dans le menu **Accueil > Vues personnalisées**,
+affichez le mode édition, sélectionnez votre vue et cliquez sur le
+bouton **Ajouter widget**.
 
-Définissez un titre pour votre widget (par exemple: Open-Tickets) et
+Définissez un titre pour votre widget (par exemple : _Open-Tickets_) et
```
Comment (cg-tw) [inline fr/alerts-notifications/ticketing.md:56]:
```suggestion
Définissez un titre pour votre widget (par exemple : **Open-Tickets**) et
```

File: fr/alerts-notifications/ticketing.md [hunk 2]
```diff
@@ -119,26 +119,26 @@ tickets ouverts :
...
+configuration fournisseur, vous pouvez les configurer dans **Lists** et
+**Custom list definition**. Pour chaque entrée dans **Lists**, vous pouvez
 définir :
 
--   **Id** : valeur alphanumeric value (doit être unique)
--   **Label** : affiché dans la fenêtre
--   **Type** : type de liste. Il existe 3 types de listes
-    -   Liste du fournisseur (données du logiciel de gestion de tickets)
-    -   Liste Centreon (comme `Host group`)
+-   **Id** : valeur alphanumerique (doit être unique) ;
```
Comment (cg-tw) [inline fr/alerts-notifications/ticketing.md:133]:
```suggestion
-   **Id** : valeur alphanumérique (doit être unique) ;
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Comment by Julien-Rozet** [comment]
How to merge this changes to main repo?

**Comment by cg-tw** [comment]
Hello,
This PR is based on our old file structure and is not compatible with the new one. You will need to create a new PR based on branch "staging".

**Pull request closed without merging**